user/util.go

```diff
@@ -2,11 +2,29 @@ package user
 
 import (
 	"crypto/ecdsa"
+	"fmt"
 
 	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
 )
 
 // IDFromKey forms the ID using script hash calculated for the given key.
 func IDFromKey(id *ID, key ecdsa.PublicKey) {
 	id.SetScriptHash((*keys.PublicKey)(&key).GetScriptHash())
 }
+
+// IDFromSigner forms the ID using script hash calculated for the given key.
+func IDFromSigner(id *ID, signer neofscrypto.Signer) error {
```

Comment:
`IDFromSigner(signer, id)`? Rationale: input first, output then. Although I'd expect it be even `func IDFromSigner(signer) (id, error)`, but not sure it's convenient for code that uses this `func`.

Reply:
Keeping as is for consistency for now.